versions/1/changelog.yml

```diff
@@ -27,7 +27,7 @@
   - description: Add `hidden` fields to data stream manifest file.
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/91
-  - description: Revert: Allow `condition` under inputs.
+  - description: 'Revert: Allow `condition` under inputs.'
```

Comment:
This was complaining in vscode because `description: Revert:` is invalid yml I think.